scripts/remove_boilerplate.py

```diff
@@ -214,6 +252,12 @@ def main():
     logging.getLogger('trafilatura').setLevel(logging.ERROR)
     install_mp_handler()
 
+    if args.token_filtering and args.token_whitelist:
+        global whitelist
+        with open(args.token_whitelist, 'rt') as list_file:
+            for token in list_file:
```

Comment:
Or shorter:
```Python
whitelist = set(map(str.strip, list_file))
```
But then you need the `global`. If you want to skip that as well, you could do
```Python
whitelist.update(map(str.strip, list_file))
```

Reply:
Thx!